src/redux/common-store/common.reducer.js

```diff
@@ -1,8 +1,10 @@
-import { SET_CART , SET_WISHLIST } from './common.types';
+import { SET_CART, SET_WISHLIST } from './common.types';
+import { getFromLocalStorage } from '../../services/local-storage.service';
+import { CART_KEY, WISHLIST_KEY } from '../../configs';
 
 export const initialState = {
-  cart: [],
-  wishlist: []
+  cart: getFromLocalStorage(CART_KEY),
```

Comment:
The initial state behind the component should be a constant. You should use the lifecycle hook if you want to set it from the local storage on the mount.

Reply:
Fixed. Now the state is updated with useEffect